app/controllers/simplified-account/settings/team-members/change-permission/change-permission.controller.js

```diff
@@ -0,0 +1,33 @@
+const { findByExternalId } = require('@services/user.service')
+const { response } = require('@utils/response')
+const formatSimplifiedAccountPathsFor = require('../../../../../utils/simplified-account/format/format-simplified-account-paths-for')
+const paths = require('@root/paths')
+
+async function get (req, res, next) {
+  const externalServiceId = req.service.externalId
```

Comment:
I think just `serviceId` is fine and more in line with usage elsewhere

Reply:
yeah good point